src/Dto/RegisterResponse.php

```diff
@@ -57,7 +57,7 @@
      * @var int
      * @see https://fidoalliance.org/specs/fido-u2f-v1.0-nfc-bt-amendment-20150514/fido-u2f-javascript-api.html#error-codes
      */
-    public $errorCode;
+    public $errorCode = self::ERROR_CODE_OK;
```

Comment:
Hmm yes, the device doesn't send an error code when there's no error. I will make this explicit in the translation of the device's response object to this DTO.